inst/automation/git_urls.rb

```diff
@@ -49,8 +49,14 @@ def gh_default_branch(owner_repo)
   res_ropenscilabs << fetch_gh('ropenscilabs', x)
 end
 
+# ropensci-archive
+# why are number of pages hard-coded? isn't this hard to maintain?
+res_ropensciarchive = []
```

Comment:
this is what I mean by hard-coded number of pages.

Reply:
just the way I did it back when I started. Instead we could do one request to get the number of repos, then paginate through that by chunks of 100

Reply:
too bad the code doesn't use crul's paginator then :wink: